addons/disposable/functions/fnc_takeLoadedATWeapon.sqf

```diff
@@ -16,19 +16,29 @@
 #include "script_component.hpp"
 
 params ["_unit"];
-TRACE_1("params",_unit);
+TRACE_1("params", _unit);
```

Comment:
macros are not commands, they are for preprocessor. Just don't use spaces if it's not intended